packages/devextreme/js/__internal/ui/form/m_form.layout_manager.utils.ts

```diff
@@ -122,8 +167,55 @@ export function convertToLabelMarkOptions({
   };
 }
 
+// eslint-disable-next-line @typescript-eslint/naming-convention
+function _getDropDownEditorOptions(
```

Comment:
Actually no need to add an underscore here while it's not a method but a function. It's clear that this function will not be used from external while it's not exported
```suggestion
function getDropDownEditorOptions(
```

Reply:
I've added underscore as many other functions (like `_convertToEditorOptions`) in this file are named that way. I thought it is like a standard